server/images.go

```diff
@@ -743,7 +691,7 @@ func CopyModel(src, dest string) error {
 	return nil
 }
 
-func deleteUnusedLayers(skipModelPath *ModelPath, deleteMap map[string]bool, dryRun bool) error {
+func deleteUnusedLayers(skipModelPath *ModelPath, deleteMap map[string]struct{}, dryRun bool) error {
```

Comment:
Why change this to a struct?

Reply:
It does clean it up slightly. The boolean wasn't really necessary because the deleteMap is just holding the keys, so the bool is redundant. The side benefit is that it also saves a marginal amount of memory.